OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/Preferences.java

```diff
@@ -131,6 +125,76 @@ public void setFirstTime(boolean value) {
         editor.commit();
     }
 
+    public boolean isUsingEncryptedKeyRings() {
+        return mSharedPreferences.getBoolean(Constants.Pref.USING_ENCRYPTED_KEYRINGS, true);
+    }
+
+    public void setUsingEncryptedKeyRings(boolean value) {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putBoolean(Constants.Pref.USING_ENCRYPTED_KEYRINGS, value);
+        editor.commit();
+    }
+
+    public boolean isAppLockReady() {
+        return mSharedPreferences.getBoolean(Pref.IS_APP_LOCK_READY, false);
+    }
+
+    public void setIsAppLockReady(boolean value) {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putBoolean(Pref.IS_APP_LOCK_READY, value);
+        editor.commit();
+    }
+
+    public boolean useApplock() {
+        return mSharedPreferences.getBoolean(Pref.USE_APPLOCK, false);
+    }
+
+    public void setUseApplock(boolean value) {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putBoolean(Pref.USE_APPLOCK, value);
+        editor.commit();
+    }
+
+    public boolean hasMasterPassphrase() {
+        return mSharedPreferences.getBoolean(Constants.Pref.HAS_MASTER_PASSPHRASE, false);
+    }
+
+    public void setHasMasterPassphrase(boolean value) {
+        SharedPreferences.Editor editor = mSharedPreferences.edit();
+        editor.putBoolean(Pref.HAS_MASTER_PASSPHRASE, value);
+        editor.commit();
+    }
+
+    public boolean isPartiallyMigrated() {
```

Comment:
Explaining what states `isPartiallyMigrated` and `isMidwayChangingPassphraseWorkflow` in the code would be nice.